openmetadata-ui/src/main/resources/ui/src/components/Task/TaskTab/TaskTab.component.tsx

```diff
@@ -162,8 +159,8 @@ export const TaskTab = ({
     updateTask(TaskOperation.RESOLVE, taskDetails?.id + '', data)
       .then(() => {
         showSuccessToast(t('server.task-resolved-successfully'));
-        rest.onUpdateEntityDetails?.();
-        history.push(getEntityLink(entityType ?? '', entityFQN ?? ''));
+        rest.onTaskFeedFetch?.();
+        rest.onUpdateTaskFilter?.('close');
```

Comment:
I feel we should club these two prop with `afterClose` prop and let parent decide what they wanna do after task closes